app/src/components/common/TmSessionImportName.vue

```diff
@@ -7,13 +7,30 @@
       <div class="session-main bottom-indent">
         <Steps :steps="[`Recover`, `Name`, `Password`]" active-step="Name" />
         <TmFormGroup field-id="import-name" field-label="Your Address">
+          <span
+            v-if="networkCryptoTypes.length > 1 && attempt > 0"
+            class="algo"
+          >
+            - {{ currentCryptoView }}
```

Comment:
```suggestion
            {{ currentCryptoView }}
```

Reply:
@jbibla can you help with styling?

Reply:
Wasn't this your styling?

Reply:
yep, but you guys complained

Reply:
@jbibla , please come to the rescue

Reply:
i think the original "unstyled" version was the strongest and simplest

Reply:
thank you :tada:  !

Reply:
maybe I have some future as a Frontender too! :v:

Reply:
I disagree but go ahead

Reply:
But what you did looks a bit strange? Do you have any suggestion? Maybe you can make some sketch?

Reply:
Let @jbibla pick anything so we can go ahead